tests/functional/kata-deploy/kata-deploy.bats

```diff
@@ -74,9 +78,15 @@ setup() {
 teardown() {
 	kubectl get runtimeclasses -o name | grep -v "kata-mshv-vm-isolation"
 
-	if [ "${KUBERNETES}" = "k3s" ]; then
+	if [ "${KUBERNETES}" = "k0s" ]; then
+		deploy_spec="-k "${repo_root_dir}/tools/packaging/kata-deploy/kata-deploy/overlays/k0s""
```

Comment:
The quoting is bogus here. The script will break if `${repo_root_dir}` contains a space. The same error already exists in the other cases. Not necessarily asking to fix them all here but at least not to propagate the bogus construct.

Reply:
Fixed, thanks!